From cbb9c769316b4d32956a2c78aa01a38b473f0cfc Mon Sep 17 00:00:00 2001
From: David Thompson <dthompson2@worcester.edu>
Date: Fri, 30 Oct 2015 08:30:43 -0400
Subject: [PATCH] xflock4: Do not override PATH with hardcoded value.

The PATH "/bin:/usr/bin" may not be a valid search path on the user's
machine.  The screen locking program may be in /usr/local/bin or
elsewhere.  Distros that do not conform to the FHS, such as GuixSD and
NixOS, will not have their executables in either location.  Thus, we
simply leave PATH alone.
---
 scripts/xflock4 | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/scripts/xflock4 b/scripts/xflock4
index ec4d05d..e7981ac 100644
--- a/scripts/xflock4
+++ b/scripts/xflock4
@@ -21,9 +21,6 @@
 #  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
 #
 
-PATH=/bin:/usr/bin
-export PATH
-
 # Lock by xscreensaver or gnome-screensaver, if a respective daemon is running
 for lock_cmd in \
     "xscreensaver-command -lock" \
-- 
2.5.0

